src/domTree.js

```diff
@@ -505,7 +505,8 @@ export class SvgNode implements VirtualNode {
     }
 
     toMarkup(): string {
-        let markup = "<svg";
+
+        let markup = `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"`;
```

Comment:
```suggestion
        let markup = `<svg xmlns="http://www.w3.org/2000/svg"`;
```
We don't use `xlink:href` in any generated SVG. We embed images only via HTML `<img>`.